### PR TITLE
Add lanes for nonroot to increase coverage

### PR DESCRIPTION
**Pull request by xpivarc**

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -451,7 +451,136 @@ presubmits:
...
+            - "/usr/local/bin/runner.sh"
+            - "/bin/sh"
+            - "-c"
+            - "automation/test.sh"
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              memory: "34Gi"
```
Comment (fgimenez) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:496]: I'm preparing a PR for decreasing the required amount of memory to `29Gi`, could you please set it in these new e2e tests?
```suggestion
              memory: "29Gi"
```

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 2]
```diff
@@ -451,7 +451,136 @@ presubmits:
...
```
Comment (fgimenez) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:539]:
```suggestion
              memory: "29Gi"
```

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 3]
```diff
@@ -451,7 +451,136 @@ presubmits:
...
+            - "/usr/local/bin/runner.sh"
+            - "/bin/sh"
+            - "-c"
+            - " automation/test.sh"
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              memory: "34Gi"
```
Comment (fgimenez) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:583]:
```suggestion
              memory: "29Gi"
```

**Review by fgimenez — COMMENTED** [review]
Looks good, only suggestions to adapt the memory requests to the upcoming changes.

**Comment by xpivarc** [comment]
> Looks good, only suggestions to adapt the memory requests to the upcoming changes.

Adjusted :+1:

**Review by fgimenez — APPROVED** [review]
/approve

Thx!

**Pull request merged**